### ISSUE TITLE
ZPArdoise - album thumbs - missing space in ALT text

### ISSUE THREAD

**Issue by Mates-K1** [ZenPhoto20/ZenPhoto20#364]
![space](https://cloud.githubusercontent.com/assets/7648991/7769101/c40b5f9a-0081-11e5-9941-268139faa59e.jpg)
